go/gen/proto/dolt/services/eventsapi/v1alpha1/client_event.pb.go

```diff
@@ -23,12 +23,13 @@
 package eventsapi
 
 import (
+	reflect "reflect"
+	sync "sync"
+
```

Comment:
Looks like the CI formatting checks are failing on this change, which is weird because it sure seems like a GH Action made this edit for you! I took a quick look at the GH Action logs, but didn't see anything obvious... I know Dustin was making some fixes there, so might just been that. Try reverting these formatting changes to the generated files and if the GH Actions apply this again, we can dig in and figure out why it's doing this (my first guess would be maybe it's formatting with an older go version; I think I saw something similar locally before).

Reply:
Rebasing history looks like it generated a new format change at the tip. Good to know that's a thing